integration_tests/artifacts/text_to_sql/questions.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "db_id": "concert_singer",
+        "query": "select distinct country from singer where age > 20",
```

Comment:
It might be nice to have tests that try a variety of DB queries, particularly if my suggestion for features over SQL operations is reflected.

Reply:
Added another example with different keywords to test.